docs/pages/blog/2020-developer-survey-results.md

```diff
@@ -0,0 +1,303 @@
+
+## Introduction
+
+Here we asked for the good, the bad and the ugly. Thankfully, it's mostly good, but we can focus
+on the things that will make it even better.
+
+### 1. How would you feel if you could no longer use Material-UI?
+
+<img src="/static/blog/2020-survey/1.png" style="display: block; margin: 0 auto;" alt="Pie chart: 73.3% Very disappointed, 21.3% somewhat disappointed, 5.4% not disappointed." />
+
+An overwhelming majority of respondents (over 94%), would be disappointed or very disappointed if
+they could no longer use Material-UI. The benchmark for "very disappointed" is 40%,
+so we're on the right track. Don't worry, Material-UI isn't going away any time soon –
+we want to focus on making even more of you "very disappointed", as it were!
+
+### 2. How likely is it that you would recommend Material-UI to a friend or colleague?
+
+<img src="/static/blog/2020-survey/2a.png" style="display: block; margin: 0 auto;" alt="Bar chart: 0.20% - 1, 0% - 2, 0.20% - 3, 0.20% - 4, 1.02% - 5, 1.97% - 6, 8.71% - 7, 22.52% - 8, 20.88% - 9, 44.29% - 10" />
+<br />
+<img src="/static/blog/2020-survey/2b.png" style="display: block; margin: 0 auto;" alt="Pie chart: 73.3% Very disappointed, 21.3% somewhat disappointed, 5.4% not disappointed." />
+
+Developers gave Material-UI a [Net Promoter Score](https://en.wikipedia.org/wiki/Net_Promoter)
+(promoters less detractors) of 61.54%. Given the NPS range of -100 to +100, a “positive” score or NPS
+above 0 is considered “good”, +50 is “Excellent”, and above 70 is considered “world class.”
+
+### 3. Who do you think would most benefit from Material-UI?
+
+This word cloud was generated with [wordclouds.com](https://wordclouds.com).
+
+<img src="/static/blog/2020-survey/3.jpg" style="display: block; margin: 0 auto;" alt="Word cloud of who would benefit most" />
+
+Developers, developers, developers! (And "people" and "teams" in general, apparently! ) This is understandable, given the
+[job role demographic](#7-which-of-the-following-best-describes-your-current-job-role)
+of the majority of respondents. We'll continue to work on the developer experience, making it
+easier to get started with, use, and customize Material-UI.
+
+### 4. What is the main benefit you receive from Material-UI?
+
+<img src="/static/blog/2020-survey/4.jpg" style="display: block; margin: 0 auto;" alt="Word cloud of the main benefit of Material-UI" />
+
+The responses variously commented on being able to focus on the business logic, faster development,
+good looking components, ease of use, design consistency, good documentation.
+We'll make sure that we continue to do more of the things that benefit you the most.
+
+### 5. How can we improve Material-UI for you?
+
+We have grouped the answers into different categories.
+The prefix corresponds to the number of times the concern was mentioned.
+It’s sorted descending, with the most important concerns first:
+
+<style>th { text-align: left; background-color: #f2f2f2; }</style>
```

Comment:
This background colour makes text nearly impossible to read in dark mode.

Reply:
We also need dark mode images.

Reply:
What about we disable the dark mode for the blog?